template-for-recording/record_screen_and_upload.bat

```diff
@@ -137,15 +137,20 @@ for /f "delims=. tokens=1-3" %%v in ("%JAVA_FULL_VERSION%") do (
     )
 )
 
+for /f "tokens=1,2 delims=-" %%a in ("%JAVA_VERSION%") do (
+  set JAVA_VERSION=%%a
```

Comment:
We should use a different variable instead of mutating the existing one

Reply:
Good call!